### PR TITLE
Cleanup README, add start of changelog

### PR DESCRIPTION
**Pull request by rosstuck**
Let me know if I'm forgetting anything. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,33 +219,43 @@ tactician:
...
+Tactician core offers a list of custom Inflectors, all of which are supported in this bundle. Assuming a class called My\App\RegisterUserCommand(), the invoked methods on the handler would be:
 
- * `tactician.handler.method_name_inflector.handle`
- * `tactician.handler.method_name_inflector.handle_class_name`
- * `tactician.handler.method_name_inflector.handle_class_name_without_suffix`
- * `tactician.handler.method_name_inflector.invoke`
+ * `tactician.handler.method_name_inflector.handle` - `handle()`
+ * `tactician.handler.method_name_inflector.handle_class_name` - `handleRegisterUserCommand()`
+ * `tactician.handler.method_name_inflector.handle_class_name_without_suffix` - `handleRegisterUser()`
+ * `tactician.handler.method_name_inflector.invoke` - `__invoke()`
```
Comment (tyx) [inline README.md]: examples are very nice additions :+1:

Comment (rosstuck) [inline README.md]: 🙇

File: README.md [hunk 2]
```diff
@@ -91,8 +87,44 @@ foo.user.register_user_handler:
...
+## Using the Command Bus 
+Create a service and inject the command bus:
+
+```yaml
+services:
+    app.your_controller:
+        class: AppBundle\Controller\YourNameController
+        arguments:
+            - '@tactician.commandbus'
+```
```
Comment (tyx) [inline README.md]: I had already forgotten how service declaration was boring :p 

god save autowiring !

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+## 1.0.0-RC1 (2017-08-03)
+
+BC Breaks:
+- PHP 7 is now required
+
+New features:
+- Allow multiple command buses, each with separated handlers and different method inflectors
+- Add security component
```
Comment (tyx) [inline CHANGELOG.md]: `s/component/middleware/` ?

File: README.md (hunk 2 above)
Comment (rosstuck) [inline README.md]: Definitely next on the list.

**Pull request merged**